src/PKSim.Core/Model/CompoundPK.cs

```diff
@@ -1,37 +1,89 @@
-﻿using OSPSuite.Utility.Collections;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
 
 namespace PKSim.Core.Model
 {
    /// <summary>
-   /// Reperesents a cache containing specific PK-Values calculated for a given compound
+   /// Represents a cache containing specific PK-Values calculated for a given compound
    /// </summary>
    public class CompoundPK
    {
+      private readonly Cache<string, QuantityPKParameter> _quantityPKParameters = new Cache<string, QuantityPKParameter>(getKey:x => x.Name);
+
+      public CompoundPK()
+      {
+         bioAvailabilities = new Cache<int, double?>();
+         ddiAucInf = new Cache<int, double?>();
+         ddiCMax = new Cache<int, double?>();
+      }
       public string CompoundName { get; set; }
```

Comment:
Move all the private stuff and properties above the constructor. This is my expectations when I look at a class to have the members and then the constructor. Here it's a bit backward